### PR TITLE
Es6 modules

### PR DESCRIPTION
**Pull request by SiansandaTamara**
## the following were implemented
- Use ES6 syntax.
- Use of import modules
- Use of script.js
- Use of lux on library time
- No linters 
- Add git flows

### CODE REVIEW

**Review by keroloussamy — COMMENTED** [review]
Hi @SiansandaTamara, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a descriptive readme file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

File: modules/bookclass.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    const deleteBtn = document.querySelectorAll('.remove');
+    deleteBtn.forEach((el) => {
+      el.addEventListener('click', (e) => {
+        const id = e.target.getAttribute('data-id');
+        BookClass.remove(id);
+        BookClass.showBooks();
+      });
+    });
+  }
+}
```
Comment (keroloussamy) [inline modules/bookclass.js]: - [ ] Your project is complete, just one thing you have to make it, is when I add a book it's successfully added and when I go to the list page I see it, but when I refresh I don't see it again. Just you need to call the show method when the page loads so when you open the page every time you see what you have saved before.

image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @SiansandaTamara, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @SiansandaTamara👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️
- ES6 syntax ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**